_emails/kalamazooMi.md

```diff
@@ -26,32 +26,29 @@ recipients:
 - SenSMcCann@senate.michigan.gov
 - jonhoadley@house.mi.gov
 subject: Regarding the funding of KDPS
-body: "Hello, \n\nI am a constituent living in Kalamazoo County and I am writing to
-  you regarding recent police excessive force at the protests against police brutality
-  as well as Kalamazoo Department of Public Safety's place in our budgets. The police's
-  behavior at the peaceful protests in early June were been aggressive, unnecessary,
-  and intentionally provocative. It's clear from all footage that the only people
-  in danger were the protesters, surprised by police cruisers speeding towards them,
-  the National Guard boxing them in, and tear gas, a tactic not allowed in war but
-  evidently perfectly fine for a medium size city's oversized police. \n\nThis kind
-  of behavior is a long standing pattern with KDPS, a police force with multiple questionable
-  in custody deaths, excessive force accusations, a documented habit of pulling over
-  predominantly black drivers and enough unnecessary lethal equipment to outfit a
-  small army. The time for hiding behind canned apologies, promises of diversity training
-  and \"community policing\" is over. It's time to reduce police spending and reallocate
-  those funds to programs that have real tangible benefits, especially to communities
-  harmed by decades of over-policing and discrimination. \n\nKalamazoo Public School
-  officials have been sounding the alarm that due to COVID-19 tax shortfalls, they
-  could be facing unprecedented cuts to per pupil funding. Yet KDPS is accepting applications
-  for new officers despite already having more employees than any other city department.
-  Public health workers in Kalamazoo have had to sew their own face masks to serve
-  consumers safely during the pandemic. Yet KDPS has a stockpile of face shields they
-  are using for public intimidation, not protection. Kalamazoo Public Safety takes
-  up 48% of the entire general fund budget for 2020. Other public services account
-  for a paltry 8% and parks and recreation a depressing 4% of budget. That is outrageous
-  and needs to change. Kalamazoo needs dollars for services that actually improve
-  and lift up our community. Dollars for education, for food assistance, for housing,
-  for people. Not policing.\n\nSincerely,\n\n[YOUR NAME]\n\n[YOUR ADDRESS]\n\n[YOUR
-  EMAIL]\n\n[YOUR PHONE NUMBER]"
+body: Hi, my name is [YOUR NAME] and I live at [YOUR ADDRESS, YOUR CITY, YOUR COUNTY]. I’m writing to let you know that I think that Kalamazoo Department of Public Safety’s use of chemical agents on peaceful protesters in late May and early June, including youth, constituted excessive force. I also think that the curfew and the presence of the National Guard in our city on June 2 did more to harm and scare us than it did to keep us safe.
```

Comment:
Here are the edits from @teddywilson’s review! You can [<kbd>Commit Suggestion</kbd>](https://help.github.com/en/github/collaborating-with-issues-and-pull-requests/incorporating-feedback-in-your-pull-request#applying-suggested-changes) automatically.

```suggestion
body: | 
  "Hi, my name is [YOUR NAME] and I live at [YOUR ADDRESS, YOUR CITY, YOUR COUNTY]. I’m writing to let you know that I think that Kalamazoo Department of Public Safety’s use of chemical agents on peaceful protesters in late May and early June, including youth, constituted excessive force. I also think that the curfew and the presence of the National Guard in our city on June 2 did more to harm and scare us than it did to keep us safe.

  This kind of behavior is a long standing pattern with KDPS, a police force with multiple questionable in custody deaths, excessive force accusations, a documented habit of pulling over predominantly black drivers, and enough unnecessary lethal equipment to outfit a small army. The time for hiding behind canned apologies, promises of diversity training and “community policing” is over. It’s time to reduce police spending and reallocate those funds to programs that have real tangible benefits, especially to communities harmed by decades of over-policing and discrimination.

  Kalamazoo Public Safety takes up 48% of the entire general fund budget for 2020. Other public services account for a paltry 8% and parks and recreation a depressing 4% of budget. That is outrageous and needs to change. Kalamazoo needs dollars for services that actually improve and lift up our community. Dollars for education, for food assistance, for housing, for people. Not policing.

  Additionally, I want to see the following actions taken by KDPS and the City Commission to hold them accountable for their violence toward our community:

  IMMEDIATELY:
  KDPS will cease the use of tear gas or other chemical weapons
  KDPS will cease the use of riot gear or other provocative dress when interacting with the public
  KDPS will wear body cams at all times, always on and unobstructed
  The Kalamazoo City Commission will order an independent investigation into KDPS response to public protests beginning May 30
  KDPS will provide a weekly public briefing on the progress of their own investigations into police conduct related to public protests beginning May 30

  WITHIN 2 WEEKS:
  KDPS will develop and ongoing plan for quarterly reporting, to the public, of all complaints against the department or its officers, and the resolution of each
  
  WITHIN 3 MONTHS:
  ALL KDPS officers who interact with the public will have participated in racial healing circle with members of the community, led by trained facilitators
  KDPS will adopt a policy requiring its officers live within the boundaries of the area the department serves

  Please keep my interests in mind as you continue to discuss these matters and work toward justice in our community.

  Sincerely,

  [YOUR NAME]"
```

Reply:
Hi there, thanks for your suggestions here.
Everything is moving very fast here, but suffice it to say we are creating a style guide that tries to articulate that we are focusing here on succinct budget-specific messaging (defunding police) and to act against the tendency for crowdsourced messaging to become ever more expansive in scope. Also trying to stay away from fundamentally reformist messaging when possible.

If you have particular issues with the current language we can discuss them but will leave as is for now.